### DISCUSSION TITLE
Add Humidity to Daily forecast

### DISCUSSION THREAD

**Discussion by xAphex** [breezy-weather/breezy-weather#1088] [Ideas]
### Describe your idea

like the title says, please add **humidity** to **daily forecast**, it exists only in hourly forecast.

and thank you for the wonderful app <3

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Please define "humidity in daily forecast", what exactly do you expect? Please keep in mind we split daily in two periods: daytime and nighttime.

**Reply by xAphex** [reply]
this is what i mean: https://i.imgur.com/CM4kfN9.png

**Reply by papjul** [reply]
This is "humidity in hourly forecast", I'm asking for a definition of "humidity in daily forecast". We don't have it, so it has to be defined. So, what would you like to see?

**Reply by papjul** [reply]
Examples of definitions:
- Hourly temperature: average temperature of the hour
- Daytime temperature: maximum temperature of the period between 06:00 and 17:59
- Nighttime temperature: minimum temperature of the period between 18:00 and 31:59

**Reply by xAphex** [reply]
all im asking for is the humidity from lowest to highest of the current week.

in hourly forecast, you get humidity by the hour.

i just would love to see the humidity of the whole week rather than only the current day.

also what do you mean by your definitions? are you talking about temperature or humidity?

**Reply by papjul** [reply]
Temperature is an example. I don't have the definition for humidity. You have to give it to me. But I seem to understand that you want:
- Weekly max humidity: max relative humidity % from the period of today 06:00 to +7 days 05:59
- Weekly min humidity: min relative humidity % from the period of today 06:00 to +7 days 05:59

We currently have no plans to offer weekly summaries like that but there are plans to expose the hourly humidity (#863) so that you could calculate it automatically by yourself.

**Reply by xAphex** [reply]
![humidity example](https://github.com/breezy-weather/breezy-weather/assets/5391237/7830a692-27d0-46b5-b866-64830a7e17bd)

i made an example of what i'm asking for, if it helps you understand my idea.

the number% shows the humidity from lowest to highest.

**Reply by papjul** [reply]
Remember the daily forecast is split into daytime period (upper part) and nighttime period (bottom part) in all other tabs with two readings. We have to keep this consistency.

The night is not necessarily drier than the day, so showing the minimum relative humidity period of the nighttime period on the bottom part and maximum relative humidity of the daytime period on the upper part is not logic to me.

And where did the dewpoint go to? Please keep the consistency with the hourly forecast: relative humidity % as a bar and dewpoint as a line
